packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -52,11 +52,21 @@ export default class AnthosTenantsService {
   }
 
   getTenantPublicIPs(serviceName) {
-    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`);
+    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`, {
+      headers: {
+        Pragma: 'no-cache',
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
+      },
+    });
   }
 
   getTenantPrivateIPs(serviceName) {
-    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/private`);
+    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/private`, {
+      headers: {
+        Pragma: 'no-cache',
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
```

Comment:
above commend from Marie applies here too.

Reply:
Taken in account.